### ISSUE TITLE
Add Observable for Set Events

### ISSUE THREAD

**Issue by iamwyza** [cyrilletuzi/angular-async-local-storage#89]
It would be nice to be able to have an observable that listens to all Set Events such that if I call `this.localstorage.set("mykey", value);` in class X, I could then have a subscriber in class Y that hears that the value in the DB has been updated/set.
```ts
class WidgetThing {
    constructor(private readonly localStorage: LocalStorage) {}
    public SetData() {
       this.localStorage.Set("MyKey", "some random data").subscribe();
    }
}

class SomeOtherClass {
   constructor(private readonly localStorage: LocalStorage) {
       this.localStorage.Events<T>("MyKey").subscribe((data: T, event: string) =>{
           console.log("something happened to MyKey!, it was " + event);
       });
   }
}
```

**Comment by cyrilletuzi** [comment]
Thanks for your interest in the lib. As stated in the README:

> When reading data, you'll only get one value: the observable is here for asynchronicity but is not meant to emit again when the stored data is changed. And it's normal: if app data change, it's the role of your app to keep track of it, not of this lib. See #16 for more context and #4 for an example.

I'll keep this issue opened as, since the other ones, I've had some thinking, and maybe it would be possible to do this without being too out of the scope of the lib.

But I currently have no time for new features (upcoming v8 is already a lot of work). So 3 possibilities:
- wait, with no guarantees,
- do a PR (*after* having agreed on API design),
- see if your company would be OK to finance this so I can include it in my work time.

**Comment by cyrilletuzi** [comment]
Available in 9.0.0-beta.2.
